Sharpnado.Presentation.Forms.iOS/Renderers/HorizontalList/iOSHorizontalListViewRenderer.cs

```diff
@@ -306,6 +315,24 @@ private void UpdateItemsSource()
             }
         }
 
+        private void RegisterCellDataTemplates()
+        {
+            HashSet<Type> templateTypes = new HashSet<Type>();
```

Comment:
I think I would prefer the same approach as the one made in the android recycler adapter: https://github.com/roubachof/Sharpnado.Presentation.Forms/blob/822f1d08b9018078c47c81c84d7f60bac5016aac/Sharpnado.Presentation.Forms.Droid/Renderers/HorizontalList/AndroidHorizontalListViewRenderer.RecycleViewAdapter.cs#L135 to rely on the DataTemplate type instead of the view model type.

Reply:
In fact you can't always rely on the ```ViewModel``` type for the ```DataTemplate```.
For example, the ```DataTemplateSelector``` I created for the silly app isn't based on ```ViewModel``` type:
https://github.com/roubachof/Xamarin-Forms-Practices/blob/master/SillyCompany.Mobile.Practices/Presentation/Views/SillyDudeTemplateSelector.cs

Reply:
Use the DataTemplate type is definitely a better approach! 👍